packages/logs/lib/models/helpers.ts

```diff
@@ -5,15 +5,19 @@ import { z } from 'zod';
 export const operationIdRegex = z.string().regex(/([0-9]|[a-zA-Z0-9]{20})/);
 
 export interface FormatMessageData {
-    account?: { id: number; name?: string };
+    account?: { id: number; name: string };
     user?: { id: number } | undefined;
-    environment?: { id: number; name?: string } | undefined;
+    environment?: { id: number; name: string } | undefined;
     connection?: { id: number; name: string } | undefined;
-    config?: { id: number; name: string } | undefined;
-    sync?: { id: string; name: string } | undefined;
+    config?: { id: number; name: string; provider: string } | undefined;
```

Comment:
should we find a better name for config which depending on the context might mean very different things?

Reply:
agreed, I just tried to limit confusion by following the DB/libs. There is something very unfortunate with bad table name and I'm not sure we did a good job of abstracting this rn. Happy to change to integration

Reply:
we overuse config so anything other than config makes more sense to me. Up to you though

Reply:
changed everything to `integration`